README.md

```diff
@@ -147,6 +147,10 @@ following attributes:
 
   * `service_yaml`: a label for a service YAML file.
     * _Note: This option will eventually be deprecated._
+  * `sample`: a path to a sample config file. There 
+  can exist multiples of this.
+  * `gapic_yaml`: path to a gapic configuration file.
```

Comment:
Can you add a sub-bullet under this one stating it's use is only for sample generation, please? Something like:

* `gapic_yaml`: path to a gapic configuration file.
  * _Note: This config is only used for sample generation._